source/common/http/conn_manager_impl.cc

```diff
@@ -610,6 +610,18 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
 
 ConnectionManagerImpl::ActiveStream::~ActiveStream() {
   stream_info_.onRequestComplete();
+  auto host_ = connection_manager_.read_callbacks_->upstreamHost();
+
+  if (host_ != nullptr) {
+    if (host_->cluster().requestResponseSizeStats().has_value()) {
+      // record request body size
+      host_->cluster().requestResponseSizeStats()->get().upstream_rq_body_size_.recordValue(
```

Comment:
suggest making a temp for readability (and slight perf benefit):
```
  ClusterRequestResponseSizeOptRef req_resp_stats = host_->cluster().requestResponseSizeStats();
  if (req_resp_stats.has_value()) {
    req_resp_stats->get().upstream_rq_body_size_.recordValue(stream_info_.bytesReceived());
...
```
similar patterns below. Actually does it work to use
```
 req_resp_stats->upstream_rq_body_size_.recordValue(stream_info_.bytesReceived());
```
omitting the `->get().` ?

Reply:
Will update.
IIRC, omitting the `->get()` doesn't work, `get()` gives access to the stored reference. Will check again.